operations/security_and_authentication/SOPS.md

```diff
@@ -0,0 +1,210 @@
+the customer managed network.
+3. A version of SOPS that supports HashiCorp Vault is installed.
+   1. [For the latest version and how to install it](https://github.com/getsops/sops/releases)
+   2. Mac environments using brew: `brew install sops`
+   3. Running SOPS from a container: `podman run ghcr.io/getsops/sops:v3.9.1-alpine`
+   4. From a distribution repository: `zypper install sops`
+
+### SOPS Use Cases
+
+These are example use cases for how SOPS can be used to secure sensitive information.
+The information provided here is a subset of what SOPS can do. The [SOPS readme](https://github.com/getsops/sops/blob/main/README.rst)
+provides more comprehensive examples of how it can be used.
+
+#### Use SOPS to interactively encrypt sensitive values that are unique to a tenant
+
+(`From a SOPS Equipped Environment`)
+
+```bash
+sops --hc-vault-transit $VAULT_ADDR new_secret_contents.yml
+```
+
+SOPS opens up an interactive text editor allowing you to create and modify a
+new file containing secret information.
+
+#### Use SOPS to encrypt an existing file
+
+(`From a SOPS Equipped Environment`)
+
+```bash
+sops encrypt --hc-vault-transit $VAULT_ADDR csm_repos.yml > csm_repos.sops.yml
+```
+
+#### Use SOPS to decrypt an already encrypted file
+
+(`From a SOPS Equipped Environment`)
+
+```bash
+sops decrypt csm_repos.sops.yml > csm_repos.decrypted.yml
+```
+
+#### Using SOPS with VCS and CFS with Ansible
+
+CFS' version of Ansible is SOPS enabled, meaning it is not necessary to store
+sensitive information in variables within VCS. This is important, as only members
+of your tenancy should have access to the secure information, and no one else.
+
+By convention, Ansible is configured to decrypt any and all `hostvars` and `groupvars`
+that are stored within VCS. This allows users to checkout the contents of their
+Ansible and git repositories, encrypt those files and variables that are considered
+sensitive, and then check in the SOPS encrypted version of these files.
+
+Ansible relies on filename hints within`hostvars` and `groupvars` in order to determine
+which files, if any, require decryption before Ansible is run. Ansible will decrypt
+any `hostvars` or `groupvars` files that include a `*.sops.*` pattern.
+
+1. [Perform a local checkout of your CFS Configuration from VCS](../configuration_management/Version_Control_Service_VCS.md)
+2. Replace sensitive information with Encrypted Versions
+
+    (`Example from a SOPS Equipped Environment with a checked out CFS Configuration`)
+
+    ```bash
+    cd csm-config
+    sops encrypt --hc-vault-transit $VAULT_ADDR user_passwords.yml > user_passwords.sops.yml
+    rm user_passwords.yml
+    git rm user_passwords.yml
+    git add user_passwords.sops.yml
+    git commit -m "Securing sensitive information in vault"
+    git push origin HEAD
+    ```
+
+3. Check in updated CSM Config and register configuration with CFS
+4. Create a CFS Session (either automatically as part of boot or through CFS API)
```

Comment:
```suggestion
1. [Perform a local checkout of your CFS Configuration from VCS](../configuration_management/Version_Control_Service_VCS.md).
2. Replace sensitive information with Encrypted Versions.

    (`Example from a SOPS Equipped Environment with a checked out CFS Configuration`)

    ```bash
    cd csm-config
    sops encrypt --hc-vault-transit $VAULT_ADDR user_passwords.yml > user_passwords.sops.yml
    rm user_passwords.yml
    git rm user_passwords.yml
    git add user_passwords.sops.yml
    git commit -m "Securing sensitive information in vault"
    git push origin HEAD
    ```

3. Check in the updated CSM configuration and register it with CFS.
4. Create a CFS Session (either automatically as part of boot or through CFS API).
```